File: api/Model/Company.cs

using System;

namespace api.Model
{
    public class Company
    {
        public Guid Id { get; set; }
        
        public int Nit { get; set; }
        
        public string CompanyName { get; set; }

        public string FirstName { get; set; }

        public string SecondName { get; set; }

        public string FirstLastName { get; set; }

        public string SecondLastName { get; set; }

        public string Email { get; set; }

        public bool AuthorizeMessagesPhone { get; set; }

        public bool AuthorizeMessagesEmail { get; set; }

        public int IdentificationTypeId { get; set; }
        
        public IdentificationType IdentificationType { get; set; }
    }
}

File: client/src/app/model/company.ts
import {IdentificationType} from "./IdentificationType";

export interface Company {
  id: string;
  nit: number;
  firstName: string;
  secondName: string;
  firstLastName: string;
  secondLastName: string;
  companyName: string;
  email: string;
  identificationType: IdentificationType;
  authorizeMessagesPhone: boolean;
  authorizeMessagesEmail: boolean;
  identificationTypeId: number;
}
